phpcs-compat.xml.dist

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<ruleset name="WooCommerce Payments PHP Compatibility">
+	<description>WooCommerce Compatibility PHP_CodeSniffer ruleset.</description>
+
+	<arg value="ps"/>
+	<arg name="parallel" value="20"/>
+
+    <!-- Exclude paths -->
+	<exclude-pattern>./dist/*</exclude-pattern>
+	<exclude-pattern>./release/*</exclude-pattern>
+	<exclude-pattern>./docker/*</exclude-pattern>
+	<exclude-pattern>./node_modules/*</exclude-pattern>
+	<exclude-pattern>./vendor/*</exclude-pattern>
+	<exclude-pattern>tests/</exclude-pattern>
+	<exclude-pattern>*\.(?!php$)</exclude-pattern>
+
+	<!-- Configs -->
+	<config name="minimum_supported_wp_version" value="7.0" />
```

Comment:
This should be the WordPress version, not the PHP version.

Ref: https://github.com/WordPress/WordPress-Coding-Standards/wiki/Customizable-sniff-properties#setting-minimum-supported-wp-version-for-all-sniffs-in-one-go-wpcs-0140

Reply:
Thank you for noticing! Somehow I missed the `wp_` prefix, but now it's changed to 5.6, the same as the `Requires WP` plugin header.